flintrock/config.yaml.template

```diff
@@ -23,6 +23,9 @@ providers:
     # vpc-id: <id>
     # subnet-id: <id>
     # placement-group: <name>
+    # security-groups:
+    #   - sg-group1
```

Comment:
I was thinking these would be security group names, not IDs. I think that's more user-friendly. Do you agree?